### ISSUE TITLE
TimeWrap should probably be TimeWarp?

### ISSUE THREAD

**Issue by Doprez** [Nicogo1705/BepuPhysicIntegrationTest#6]
![image](https://github.com/Nicogo1705/BepuPhysicIntegrationTest/assets/73259914/4407c9cd-43b8-4906-b48e-10309057a5b5)
Im fairly certain `TimeWrap` should be `TimeWarp` but just wanted confirmation before I change it.

**Comment by Nicogo1705** [comment]
Indeed, I thought it was "wrap," but it is indeed "warp." in English.

**Comment by Nicogo1705** [comment]
fixed in lasts PR.
